src/groups/bmq/bmqt/bmqt_subscription.h

```diff
@@ -203,9 +203,19 @@ class Subscription {
 
     /// Create a new Subscription by copying values from the specified
     /// `other`.
-    Subscription(const Subscription& other);
+    explicit Subscription(const Subscription& other);
```

Comment:
We shouldn't need this `explicit here`, just the assignment operator

Reply:
My bad